app.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const app = express();
+
+// const {  } = require("./controller/controllers");
+
+const endpoints = require("./endpoints.json");
+
+app.use(express.json());
+
+app.get("/api", (req, res) => {
+  res.status(200).send( { endpoints } );
```

Comment:
This should be handled in the controller, utilizing the MVC pattern we learned about :) For this one, you don't necessarily need the model, but responses are always handled by the controller.